server.js

```diff
@@ -0,0 +1,59 @@
+const express = require('express');
+const request = require('request');
+
+const port = 5000;
+
+global.access_token = '';
+
+const spotify_client_id = 'd63fc97ff8f84a7bb58d1d9cf73ecfed';
+const spotify_client_secret = 'a2d83594e53642be961588d675d12828';
+const spotify_redirect_uri = 'http://localhost:3000/auth/callback';
+
+let app = express();
+
+app.get('/auth/login', (req, res) => {
+  let scope = 'streaming user-read-email user-read-private';
+
+  let searchParams = new URLSearchParams({
```

Comment:
both `scope` and `searchParams` are never redefined. You should use `const` here instead